frontend/src/components/Calendar/Calendar.module.css

```diff
@@ -79,17 +86,17 @@
 
 .day {
     padding: 2px; 
-    border: 1px solid #880808; 
+    border: 1px solid #CF0A2C; 
```

Comment:
If this is the class to edit the calendar box days, perhaps change the background color inside of the boxes. Maybe to white or grey?

you could do this by adding this line "background-color: white;" to change it to white

or this line "background-color: #f0f0f0;" to change it to grey.

Personally, it looks weird having the background of the boxes be the same as the background color of the website. So changing it to white or grey could help to make the calendar pop out.

Reply:
Hi Aarush, you make a good point about the background color of the website as of right now not blending in well with the calendar boxes. Although I believe that it would be fine as it is now if the background color of the website globally was switched to white or any white-variation kind of color instead of black.

Example:
![image](https://github.com/user-attachments/assets/90d690a5-826e-4e40-a6fd-793760d7e226)

I switched the website to have a white background and kept the boxes to have the same background color as the website. In my opinion, it looks better like this. Keep in mind that the LongTitle... event is a bright green and so it looks awkward on a white background, however I do not intend to have that bright of a color to be an option for when a user chooses their event color when I set up my CRUD operations for the event. Let me know of what you think of the design in the screenshot above.

- Nathan Nguyen

Reply:
Either the background color of the website and the background color of the boxes are the same, or maybe the boxes are slightly different in color than the website. Either option I think is the best.

Reply:
> Hi Aarush, you make a good point about the background color of the website as of right now not blending in well with the calendar boxes. Although I believe that it would be fine as it is now if the background color of the website globally was switched to white or any white-variation kind of color instead of black.
> 
> Example: ![image](https://private-user-images.githubusercontent.com/50503296/380040917-90d690a5-826e-4e40-a6fd-793760d7e226.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk4ODYzNjMsIm5iZiI6MTcyOTg4NjA2MywicGF0aCI6Ii81MDUwMzI5Ni8zODAwNDA5MTctOTBkNjkwYTUtODI2ZS00ZTQwLWE2ZmQtNzkzNzYwZDdlMjI2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI1VDE5NTQyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlhMWZhMjA1YzFkYTZhNWMyMjZkMzg0NTQ3OWIwODk1ZjYxNjJiN2Y5NjI0NDZhM2I4YjRiZDI2NDY4M2EzOGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.YoMR6sH69hhqPdhwM8C6x5Q3ENEj2lILOKifxtKAhg8)
> 
> I switched the website to have a white background (#f2f2f2) and kept the boxes to have the same background color as the website. In my opinion, it looks better like this. Keep in mind that the LongTitle... event is a bright green and so it looks awkward on a white background, however I do not intend to have that bright of a color to be an option for when a user chooses their event color when I set up my CRUD operations for the events. Let me know of what you think of the design in the screenshot above.
> 
> * Nathan Nguyen

Hey Nathan, I started messing around with the calendar's coding to understand it better. I was able to make a page that looks like this:

![s2](https://github.com/user-attachments/assets/0b628f4c-a627-484f-b0d2-8e1ae0a78c26)

Again, this is just a nitpick, but if you want, I can tell you what you need to add or change. It's pretty simple, so I'll just add comments on the "Files Changed" page.

Also, I noticed in your globals.css file, you have that ":root" on line 3. As you can see on line 2 from this image below, I think you can delete lines 3 to 6. 
![q2](https://github.com/user-attachments/assets/dd151da0-5a37-439b-a0be-5d8c181d8cff)

I tried changing the background and foreground colors with that first ":root" but it seems to not be working. But if you see on line 10, I changed the color to #a9a9a9 (Medium Grey) for the background of  the website